roles/install-nfs/tasks/main.yaml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Install nfs server
+  yum:
+    name: nfs-utils
+
+- name: Create nfs share folder
+  file:
+    path: "{{ nfs_directory }}"
+    state: directory
+    mode: 0755
+    recurse: yes
+
+- name: Add shared directory in exports file
+  ansible.builtin.lineinfile:
+    path: /etc/exports
+    regexp: "{{ nfs_directory }}"
+    line: "{{ nfs_directory }} *(rw,sync,no_root_squash)"
+    state: present
+
+- name: Start services
+  block:
+    - name: Start rpcbind
+      systemd:
+        name: rpcbind
+        enabled: yes
+        state: restarted
+
+    - name: Start nfs-server
```

Comment:
I was wondering if you want to server nfs-server also part of pod?!

Reply:
something similar to this - https://github.com/appscode/third-party-tools/blob/master/storage/nfs/README.md

Reply:
The NFS server is set up on the master node, while the provisioner pod in Kubernetes acts as the client.